### PR TITLE
Feat/home refresh

### PR DESCRIPTION
**Pull request by alliefleming**
New homepage layout. Main navigation changes.

### CODE REVIEW

File: about/faq.html [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
+  id: faq
   site:
-    highlight: membership
+    highlight: about
   side:
-    id: membership
-    highlight: membership-faq
+    id: about
+    highlight: faq
+redirect_from: "/membership/faq"
```
Comment (brandonarbini) [inline about/faq.html]: Needs a trailing slash

File: spaces.markdown [hunk 2]
```diff
@@ -17,8 +17,8 @@ redirect_from:
 - "/workspace/"
 - "/workspaces/"
 - "/locations/"
+- "/membership/access-to-spaces"
```
Comment (brandonarbini) [inline spaces.markdown]: Needs a trailing slash

File: _assets/stylesheets/_base-layout.scss [hunk 3]
```diff
@@ -1,19 +1,19 @@
...
-  @include media-breakpoint-only(md) {
-    background-image: url("/assets/images/header-bg-md.svg");
-  }
-  @include media-breakpoint-only(sm) {
-    background-image: url("/assets/images/header-bg-sm.svg");
-  }
-  @include media-breakpoint-only(xs) {
-    background-image: url("/assets/images/header-bg-xs.svg");
-  }
+  // background-repeat: no-repeat;
```
Comment (brandonarbini) [inline _assets/stylesheets/_base-layout.scss]: What's the deal with this? </seinfeld>

Comment (alliefleming) [inline _assets/stylesheets/_base-layout.scss]: ahhh, whoops. now deleted. removed top header diagonal now that we have feature banner on home page.

**Pull request merged**